pyomo/solvers/tests/models/LP_simple.py

```diff
@@ -8,7 +8,7 @@
 #  This software is distributed under the 3-clause BSD License.
 #  ___________________________________________________________________________
 
-import pyomo.kernel as pmo
```

Comment:
I think we should keep this import as `pmo` to avoid confusion between AML and kernel components

Reply:
I want you to know - after doing a million of these reversions, I'm getting annoyed. Grrrr.